src/relayserver/RelayServer.ts

```diff
@@ -86,9 +88,12 @@ export interface RelayServerParams {
   readonly baseRelayFee: number | undefined
   readonly pctRelayFee: number | undefined
   readonly gasPriceFactor: number
+  readonly registrationBlockRate?: number
   readonly url?: string
   readonly workerMinBalance: number | undefined // = defaultWorkerMinBalance,
   readonly workerTargetBalance: number | undefined // = defaultWorkerTargetBalance,
+  readonly managerTargetBalance: number | undefined // = defaultManagerTargetBalance,
+  readonly minHubWithdrawalBalance: number | undefined // = defaultManagerTargetBalance,
```

Comment:
You use  '?:' and '| undefined' interchangeably. I think these are equivalent and it's better to stick to one syntax in general...

Reply:
Nothing changed here

Reply:
This is true in general in the code, since it's not only me writing it. Consistency is important but should be fixed in another PR imo.